gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Project-wide Gradle settings.
-VERSION_NAME=5.1.0-SNAPSHOT
+VERSION_NAME=5.2.0-SNAPSHOT
```

Comment:
Are there any API changes?

Reply:
Nope, just don't want to mix it with previous version. 
Should we make it 5.1.1?

Reply:
`5.1.0` has not been released yet, so we can keep using that snapshot version.

Reply:
yeah, but we may face some caching problems because of the same naming. I know that it's still 'SNAPSHOT'. 
Do you suggest leaving  5.1.0 as it is?

Reply:
yes, we should leave it at `5.1.0`